src/webviews/extension-side/serviceRegistry.node.ts

```diff
@@ -1,67 +1,67 @@
-import { PlotViewer } from './plotting/plotViewer.node';
-import { PlotViewerProvider } from './plotting/plotViewerProvider.node';
-import { IPlotViewer, IPlotViewerProvider } from './plotting/types';
-import { PlotSaveHandler } from './plotView/plotSaveHandler.node';
-import { PlotViewHandler } from './plotView/plotViewHandler.node';
-import { RendererCommunication } from './plotView/rendererCommunication.node';
-import { NotebookWatcher } from './variablesView/notebookWatcher';
-import { INotebookWatcher, IVariableViewProvider } from './variablesView/types';
-import { VariableViewActivationService } from './variablesView/variableViewActivationService';
-import { VariableViewProvider } from './variablesView/variableViewProvider';
-
-export function registerTypes(serviceManager: IServiceManager) {
-    serviceManager.addSingleton<IExtensionSingleActivationService>(
-        IExtensionSyncActivationService,
-        RendererCommunication
-    );
-
-    // Data Viewer
-    serviceManager.add<IDataViewer>(IDataViewer, DataViewer);
-    serviceManager.addSingleton<IDataViewerFactory>(IDataViewerFactory, DataViewerFactory);
-    serviceManager.addSingleton<IDataViewerDependencyService>(
-        IDataViewerDependencyService,
-        DataViewerDependencyService
-    );
-    serviceManager.addSingleton<IExtensionSingleActivationService>(
-        IExtensionSingleActivationService,
-        DataViewerCommandRegistry
-    );
-
-    // Plot Viewer
-    serviceManager.add<IPlotViewer>(IPlotViewer, PlotViewer);
-    serviceManager.addSingleton<IPlotViewerProvider>(IPlotViewerProvider, PlotViewerProvider);
-    serviceManager.addSingleton<PlotSaveHandler>(PlotSaveHandler, PlotSaveHandler);
-    serviceManager.addSingleton<PlotViewHandler>(PlotViewHandler, PlotViewHandler);
-
-    // Variable View
-    serviceManager.addSingleton<INotebookWatcher>(INotebookWatcher, NotebookWatcher);
-    serviceManager.addSingleton<IExtensionSingleActivationService>(
-        IExtensionSingleActivationService,
-        VariableViewActivationService
-    );
-    serviceManager.addSingleton<IVariableViewProvider>(IVariableViewProvider, VariableViewProvider);
-    serviceManager.add<IJupyterVariableDataProvider>(IJupyterVariableDataProvider, JupyterVariableDataProvider);
-    serviceManager.addSingleton<IJupyterVariableDataProviderFactory>(
-        IJupyterVariableDataProviderFactory,
-        JupyterVariableDataProviderFactory
-    );
-}
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+'use strict';
+
+import { IExtensionSingleActivationService, IExtensionSyncActivationService } from '../../platform/activation/types';
+import { IServiceManager } from '../../platform/ioc/types';
+import { DataViewer } from './dataviewer/dataViewer';
+import { DataViewerCommandRegistry } from './dataviewer/dataViewerCommandRegistry';
+import { DataViewerDependencyService } from './dataviewer/dataViewerDependencyService';
```

Comment:
This file looks like it was completely rewritten unless you hide the whitespaces. I'm guessing it has something to do with the line breaks. Anyway, to hide the whitespaces, you can click here: https://github.com/microsoft/vscode-jupyter/pull/10545/files?diff=unified&w=1

Reply:
Do you mean `npm run prettier-fix`? I am running it and I see no difference.

Reply:
I've been using wsl, mac, Linux and windows and never had any such issues.
i believe ian is on a mac and rich has used wsl as well.

not sure we need such a change.

Reply:
so, the underlying problem is that the line endings have changed -- not sure what else to propose

Reply:
I changed my .gitconfigs and made sure to undo these changes 👍 though for the record, I do prefer LF ends of lines, just like I prefer formatting enforced 😆